tests/fixtures/commands/drupal-services-setup.yml

```diff
@@ -0,0 +1,46 @@
+- config:
+    runner: |
+      my_settings:
+        foo: bar
+      drupal:
+        service_parameters:
+          settings:
+            foo: ${my_settings.foo}
+  expected:
+    - contains: |
+        parameters:
+          settings:
+            foo: bar
+- config:
+    force: true
+    runner: |
+        my_settings:
+          foo: bar
+        drupal:
+          service_parameters:
+            settings:
+              foo: ${my_settings.foo}
+  expected:
+    - contains: |
+        parameters:
+          settings:
+            foo: bar
+- config:
+    force: false
```

Comment:
`force: false` is not a thing, :smile: it's either there or it's not. If you want to test it for false then don't set it.

Reply:
Done.